test/integration/targets/nxos_bgp/tests/cli/sanity.yaml

```diff
@@ -0,0 +1,135 @@
+---
+- debug: msg="START TRANSPORT:CLI nxos_bgp sanity test"
+
+- set_fact: log_neighbor_changes="true"
```

Comment:
The comments I made here are gone, and I'm going to assume that this was my fault, because I've been looking at this too long and can't remember what is what anymore.

`log_neighbor_changes` is *not* an issue with nxos_bgp (but it _is_ with nxos_bgp_neighbor). Instead, what needs to be turned off here in nxos_bgp (and its twin in nxapi) is `neighbor_down_fib_accelerate`. Sorry for the confusion